src/MouseAdapter.js

```diff
@@ -309,20 +383,196 @@ define(function(require) {
+         * observable if desired in the future.
+         * @param {object} event
+         * @private
+         */
+        _dispatchUserReScroll: function(event) {
             this._dispatchMouseWheelEnd(event);
+            this._dispatchMouseWheelStart(event);
+        },
+
+        /**
+         * Detect when the user performs a new scroll gesture.
+         * @param {object} normalizedEvent
+         * @private
+         */
+        _detectUserScroll: function(normalizedEvent) {
+
+            /*
+             * This is a helper function which takes in a set of consecutive linear data 
+             * and tries to determine whether it is increasing or decreasing.  It can
+             * operate on an entire array, or only a specified section of it. When given a
+             * starting point and a length that extends outside the boundaries of the array,
+             * the function assumes the data wraps around to the beginning of the array.
+             * @param {Array.<number>} array The array of data to perform analysis on.
+             * @param {number} arrayMaxLength [optional] The max length of the array.  If
+             * not given, assumed to be array.length
+             * @param {number} startAt [optional] The index in the array where the target 
+             * data begins.  If not specified, assumes this index is 0.
+             * @param {number} length [optional] The number of data points to examine.  
+             * If not specified, assumed to be the entire array length.
+             * @return {number} Returns:
+                    1 if data appears to be increasing.
+                   -1 if data appears to be decreasing.
+                    0 if there is no discernable trend.
+             */
+            var calculateDeltaDirection = function(array,startAt,length,arrayMaxLength) {
+                arrayMaxLength = arrayMaxLength || array.length;
+                startAt = startAt || 0;
+                length = length || arrayMaxLength;
+                var total = 0;
+                var offsetIdx1,offsetIdx2;
+                for (var idx1 = 0; idx1 < length; idx1++) {
+                    for (var idx2 = idx1+1; idx2 < length; idx2++) {
+                        offsetIdx1 = (idx1+startAt) % arrayMaxLength;
+                        offsetIdx2 = (idx2+startAt) % arrayMaxLength;
+                        if (array[offsetIdx1] < array[offsetIdx2]) {
+                            total += (1/(idx2-idx1));
+                        }
+                        if (array[offsetIdx1] > array[offsetIdx2]) {
+                            total += (-1/(idx2-idx1));
+                        }
+                    }
+                }
+                if (total >= 1) {
+                    return 1; // Increasing trend
+                }
+                if (total <= -1) {
+                    return -1; // Decreasing trend
+                }
+                return 0; // Stable
```

Comment:
I thought of doing it that way initially, but the actual range of values for `total` is linearly dependent on the length of the data, and not dependent at all on the magnitude of the data.  For example:

calculateDeltaDirection([1,2,3,4]) = 4.333333333333333
calculateDeltaDirection([-1,-2,-3,-4,-5,-6]) = -8.7
calculateDeltaDirection([1,2,3,4,5,6]) = 8.7
calculateDeltaDirection([1,12,103,1004,10005,100006]) = 8.7

For these reasons, i'm dont think the value of `total` has any "meaning" beyond whether it's positive or negative, and i found that bounding output to -1, 0 or 1 made using it easier.

Given all that, if you still think there's some value to returning total, i can totally refactor it.